### DISCUSSION TITLE
Static linear analysis of the beam. Executioner parameters

### DISCUSSION THREAD

**Discussion by andrey1887** [idaholab/moose#26190] [Q&A Modules: Solid mechanics] [answered]
Hello. I looked through beam analysis examples of TensorMechanics module [https://mooseframework.inl.gov/modules/tensor_mechanics/beam_vandv.html](https://mooseframework.inl.gov/modules/tensor_mechanics/beam_vandv.html).
Help me please to understand: why  `Transient` type and `NEWTON` solve_type are used.
If i need to do static analysis for linear problem maybe more appropriate types should be used?

**Comment by GiudGiud** [comment]
Hello

Transient is often used in analysis to simulate loading steps. If the time derivative is not added to the kernels, it s equivalent to a steady solve on every step

Guillaume

**Reply by andrey1887** [reply]
Thank you @GiudGiud, and what about `solve_type`?

**Reply by GiudGiud** [reply, marked as answer]
solve_type of Newton makes sense if the problem is nonlinear.
If it s truly linear, you can use a LINEAR solve_type

**Reply by andrey1887** [reply]
Thank you.
